source/main/physics/Beam.cpp

```diff
@@ -3262,7 +3262,7 @@ void Actor::ToggleTies(int group)
                             continue;
 
                         //skip if tienode is ropable too (no hk_selflock)
-                        if (itr->node->id == it->ti_beam->p1->id)
+                        if (itr->node->pos == it->ti_beam->p1->pos)

```

Comment:
I'm not entirely sure, but it seems as if it wouldn't matter if we use `id` or `pos` here.

Reply:
I can't figure this out..
`it` is a local tie.
`itr` is a neighbour ropable.
`if (itr->node->pos == it->ti_beam->p1->pos)` - makes no sense, we compare indices of neighbour ropable node and our tie-beam p1. But those live in different arrays. Tie has p1 always on local truck, p2 attaches to the neighbour.

Reply:
✔️